packages/editable-commands/README.md

```diff
@@ -0,0 +1,120 @@
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire-banner.png)
+
+# @sapphire/plugin-editable-commands
+
+**Plugin for <a href="https://github.com/sapphiredev/framework">@sapphire/framework</a> to have editable commands.**
+
+[![GitHub](https://img.shields.io/github/license/sapphiredev/plugins)](https://github.com/sapphiredev/plugins/blob/main/LICENSE.md)
+[![codecov](https://codecov.io/gh/sapphiredev/plugins/branch/main/graph/badge.svg?token=QWL8FB16BR)](https://codecov.io/gh/sapphiredev/plugins)
+[![npm bundle size](https://img.shields.io/bundlephobia/min/@sapphire/plugin-editable-commands?logo=webpack&style=flat-square)](https://bundlephobia.com/result?p=@sapphire/plugin-editable-commands)
+[![npm](https://img.shields.io/npm/v/@sapphire/plugin-editable-commands?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/plugin-editable-commands)
+[![Depfu](https://badges.depfu.com/badges/11bbf7392987e6fd51fc6559e1d42dfc/count.svg)](https://depfu.com/github/sapphiredev/plugins?project_id=15201)
+
+</div>
+
+## Description
+
+TBD.
+
+## Features
+
+-   Fully ready for TypeScript!
+-   Includes ESM ready entrypoint
+-   Full editable commands, attachments included!
+
+## Installation
+
+`@sapphire/plugin-editable-commands` depends on the following packages. Be sure to install these along with this package!
+
+-   [`@sapphire/framework`](https://www.npmjs.com/package/@sapphire/framework)
+
+You can use the following command to install this package, or replace `npm install` with your package manager of choice.
+
+```sh
+npm install @sapphire/plugin-editable-commands @sapphire/framework
+```
+
+---
+
+## Usage
+
+In your main or setup file, register the plugin:
+
+```typescript
+import '@sapphire/plugin-editable-commands/register';
+```
+
+Then use `send` or `reply` from the package, as shown below:
+
+```typescript
+import { ApplyOptions } from '@sapphire/decorators';
+import { Command, CommandOptions } from '@sapphire/framework';
+import { Message, MessageEmbed } from 'discord.js';
+import { send } from '@sapphire/plugin-editable-commands';
+
+@ApplyOptions<CommandOptions>({
+	description: 'A very cool command',
+	requiredClientPermissions: ['EMBED_LINKS']
+})
+export class UserCommand extends Command {
+	public run(message: Message) {
+		const embed = new MessageEmbed()
+			.setURL('https://github.com/sapphiredev/plugins')
+			.setColor('#7586D8')
+			.setDescription('Example description')
+			.setTimestamp();
+
+		return send(message, { embeds: [embed] });
+	}
+}
+```
+
+## Editable Commands Documentation
+
+For the full @sapphire/plugin-editable-commands documentation please refer to the TypeDoc generated [documentation](https://sapphiredev.github.io/plugins/modules/_sapphire_plugin_editable-commands.html).
```

Comment:
double check that url by running `yarn docs`, opening it with http-server or something and checking the path.